modules/claims_api/lib/custom_error.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class CustomError
+    def initialize(error, claim, method)
+      @error = error
+      @claim = claim
+      @method = method
+    end
+
+    def build_error
+      custom_error = if @error == Faraday::ConnectionFailed || @error == Faraday::ParsingError
+                       { 'messages' => [{ 'key' => 'ServiceException', 'detail' => 'test detail', status: '500' }] }
```

Comment:
`'test detail'` will need to be changed. this message is relayed to the consumer.